### PR TITLE
Writing Program: Authoring process

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabhattacharjee@Debopriyas-MacBook-Air.local>

**Description**

Screenshot 2022-06-25 at 3 49 40 AM

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by debo19** [comment]
@leecalcote this works?

File: src/sections/Community/Handbook/writing-program.js [hunk 1]
```diff
@@ -164,33 +164,46 @@ const Writers = () => {
...
-              <h2 className="modal-heading">Layer5 Content Form</h2>
-              <div className="content-form">
-                <iframe
-                  src="https://docs.google.com/forms/d/e/1FAIpQLSf0BlMvmeEVsy4x6fMdI2cWcsnyVRSCv7b8e0Ios1hEopjOZQ/viewform"
-                  loading="lazy"
-                  className="form-frame"
-                />
+            <div className="process">
+              <div className="item">
+                <p>If you’re interested in the Layer5 Writing Program, please apply. Fill out the Content Form to get started.</p>
```
Comment (leecalcote) [inline src/sections/Community/Handbook/writing-program.js:170]:
```suggestion
                <p>If you’re interested in the Layer5 Writing Program, please submit your information (and proposed topic, if you have one) to get started. <i>Don't have a topic? Don't worry have plenty of suggestions for you.</i></p>
```

Comment (leecalcote) [reply src/sections/Community/Handbook/writing-program.js:170]: This phrasing is slightly less foreboding.

File: src/sections/Community/Handbook/writing-program.js [hunk 2]
```diff
@@ -164,33 +164,46 @@ const Writers = () => {
...
+                </Modal> 
+              </div>
+              <div className="item">
+                <p>Upon being accepted, you'll work with a coordinator on the content team to decide on a topic, develop a project plan, and discuss submission details.</p>
               </div>
-            </Modal>
-            <p>Follow the instructions to contribute to your choice of content. Upon being accepted, you'll work with a coordinator on the content team to decide on a topic, develop a project plan, and discuss submission details.</p>
+              <div className="item">
+                <p>Follow the instructions to contribute to your choice of content.</p> 
+              </div>
```
Comment (leecalcote) [inline src/sections/Community/Handbook/writing-program.js:197]:
```suggestion
```

Comment (leecalcote) [reply src/sections/Community/Handbook/writing-program.js:197]: We can probably skip this last step.

**Review by leecalcote — APPROVED** [review]

File: src/sections/Community/Handbook/writing-program.js [hunk 3]
```diff
@@ -164,33 +164,43 @@ const Writers = () => {
...
-              <h2 className="modal-heading">Layer5 Content Form</h2>
-              <div className="content-form">
-                <iframe
-                  src="https://docs.google.com/forms/d/e/1FAIpQLSf0BlMvmeEVsy4x6fMdI2cWcsnyVRSCv7b8e0Ios1hEopjOZQ/viewform"
-                  loading="lazy"
-                  className="form-frame"
-                />
+            <div className="process">
+              <div className="item">
+                <p>If you’re interested in the Layer5 Writing Program, please submit your information (and proposed topic, if you have one) to get started. <i>Don't have a topic? Don't worry have plenty of suggestions for you.</i></p>
```
Comment (debo19) [inline src/sections/Community/Handbook/writing-program.js:170]:
```suggestion
                <p>If you’re interested in the Layer5 Writing Program, please submit your information (and proposed topic, if you have one) to get started. <i>Don't have a topic? Don't worry, we have plenty of suggestions for you.</i></p>
```

**Comment by l5io** [comment]
🚀 Preview for commit 8f201d64ddfc5d1ab984a60cbeebefc2946c8c50 at: https://62b63fca232d3400b29e9ade--layer5.netlify.app

**Pull request merged**